core/trino-main/src/main/java/io/trino/server/security/PasswordAuthenticatorManager.java

```diff
@@ -34,7 +34,7 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkState;
 import static com.google.common.base.Strings.isNullOrEmpty;
-import static io.airlift.configuration.ConfigurationLoader.loadPropertiesFrom;
```

Comment:
Can you please post this PR against https://github.com/airlift/airlift instead? Also can you please create a bug issue in github that explains the issue?

Reply:
Okay. I'll post this PR at airlift and create a issue.